### PR TITLE
zh-translation: content/zh/news/releases/1.4.x/announcing-1.4/upgrade-notes/index.md

### PR DESCRIPTION
**Pull request by dayuoba**
ref: https://github.com/servicemesher/istio-official-translation/issues/1588

### CODE REVIEW

File: content/zh/news/releases/1.4.x/announcing-1.4/upgrade-notes/index.md [hunk 1]
```diff
@@ -1,45 +1,42 @@
...
 
-To help with your upgrade, here are some steps you could take:
+为了帮助你升级，这里有一些你可以参照的步骤：
 
-* After upgrading Istio, run your Istio configurations with `kubectl apply --dry-run` so that you are able to know if the configurations can be accepted by the API server as well as any possible unknown and/or invalid fields to the API server. (`DryRun` feature is on by default for Kubernetes 1.13+ clusters.)
-* Use the [reference documentation](/zh/docs/reference/config/) to confirm and correct the field names and data types.
-* In addition to structural validation, you can also use `istioctl x analyze` to help you detect other potential issues with your Istio configurations. Refer to [here](/zh/docs/ops/diagnostic-tools/istioctl-analyze/) for more details.
+* 升级 Istio 之后，用 `kubectl apply --dry-run` 来跑一遍你的 Istio 配置，这样你就可以知道这些配置对于 API server 来说是不是 unknown 或者无效的字段。（`DryRun` 特性是 Kubernetes 1.13+ 版本开始的默认选项）
+* 查看 [reference documentation](/zh/docs/reference/config/) 来确认修正字段名和数据类型。
+* 另外，为了能够结构化验证，你也可以使用 `istioctl x analyze` 来帮助检查到你的 Istio 配置的潜在问题。参考 [here](/zh/docs/ops/diagnostic-tools/istioctl-analyze/) 或者更多细节。
```
Comment (gorda) [inline content/zh/news/releases/1.4.x/announcing-1.4/upgrade-notes/index.md]: > 或者更多细节。

建议：获取更多细节。

Comment (dayuoba) [reply content/zh/news/releases/1.4.x/announcing-1.4/upgrade-notes/index.md]: 嗯，修正了

**Pull request merged**